wiki/BBCode/pl.md

```diff
@@ -0,0 +1,380 @@
+  - Atrybut `LINK` nie powinien znajdować się w cudzysłowach.
+- Składnia:
+  - `LINK` to łącze.
+  - `TEXT` to tekst, który zostanie podstawiony za łącze.
+```
+[url=LINK]TEXT[/url]
+```
+
+### Profil
+
+Dodaje łącze do profilu użytkownika.
+
+Używanie tego znacznika jest **niezalecane**! Znacznik ten jest problematyczny, ponieważ istnieje możliwość, że użytkownik zmieni swój pseudonim, co sprawi, że łącze przestanie działać.
+
+Zaleca się używanie znacznika [Łącze](#łącze) używając numeru ID użytkownika zamiast znacznika Profil.
+
+- Dedykowany przycisk: Brak
+- Ważne:
+  - Używanie tego znacznika jest niezalecane!
+  - Atrybut `USER` musi być podany.
+- Składnia:
+  - `USER` to nazwa użytkownika.
+```
+[profile]USER[/profile]
+```
+
+### Google
+
+Wyszuka podany tekst używając wyszukiwarki Google.
+
+- Dedykowany przycisk: Brak
+- Ważne:
+  - Warto zanotować, że wyniki wyszukiwania nie będą takie same dla wszystkich.
+    - Część wyników będzie ukryta bazując na języku i lokalizacji.
+- Składnia:
+```
+[google]...[/google]
+```
+
+### Szczęsliwy traf
+
+Dodaje łącze do strony wyszukanej dzięki funkcji *Szczęśliwy traf* wyszukiwarki Google.
+
+
+- Dedykowany przycisk: Brak
+- Ważne:
+  - Warto zanotować, że wyniki wyszukiwania nie będą takie same dla wszystkich.
+    - Część wyników będzie ukryta bazując na języku i lokalizacji.
+- Składnia:
+```
+[lucky]...[/lucky]
+```
+
+### Lista
+
+Zmienia tekst w listę.
+
+W BBCode, listy składają się z dwóch części: znaczników - ukazujących zasięg listy - oraz punktów.
+
+- Dedykowany przycisk:
+  - Lista punktowana: ![Przycisk dla listy puntkowanej](img/list.png)
+  - Lista numerowana: ![Przycisk dla listy numerowanej](img/list-numbered.png)
+- Ważne:
+  - Wszystkie punkty muszą znajdować się wewnątrz znacznika Lista.
+  - Istnieje 6 argumentów dla atrybutu `TYPE`:
+    - _(pusty)_ - punkty
+    - `1` - liczby
+    - `a` - litery (małe litery)
+    - `A` - litery (duże litery)
+    - `i` - liczby rzymskie (małe litery)
+    - `I` - liczby rzymskie (duże litery)
+  - Listy umieszczone wewnątrz list mogą być problematyczne.
+- Składnia:
+  - `TYPE` to jeden z argumentów opisanych powyżej.
+  - Jeżeli `TYPE` nie jest określony, zostanie użyta lista punktowana.
+```
+[list=TYPE]
+[*]...
+[/list]
+```
+
+### Obrazek
+
+Dodaje obrazek z łącza.
+
+Obrazki mogą być załączane z każdej strony, dopóki znajdują się one w łączach.
+
+**Nie dodawaj obrazków używając lokalnej ścieżki dostępu!** Użycie `C:\Users\Name\Pictures\image.jpg` **nie zadziała**.
+
+Prosimy o dodawanie zdjęć z wiarygodnych stron internetowych takich jak [imgur](https://imgur.com/ "imgur") czy [puush](https://puush.me/ "puush"). Gdy zdjęcie zostanie przesłane, skopiuj bezpośredni odnośnik do obrazka i wstaw je między znacznik otwierający a zamykający. Miej na uwadze, że niektóre strony nie podają bezpośrednich odnośników do zdjęć. Strony, takie jak te podane powyżej, podają owe odnośniki, ponieważ są to strony oferujące darmowy hosting grafiki.
```

Comment:
Prosimy o dodawanie obrazów z zaufanych stron internetowych, takich jak [imgur](https://imgur.com/ "imgur") czy [puush](https://puush.me/ "puush"). Gdy obraz zostanie przesłany, skopiuj bezpośredni odnośnik do obrazka i wstaw je między znacznik otwierający a zamykający. Miej na uwadze, że niektóre strony nie podają bezpośrednich odnośników do obrazów (tzw. hotlinków). Mimo to, większość z nich nie ma trudności z podawaniem bezpośrednich odnośników, ponieważ opierają się one na oferowaniu darmowego hostingu grafiki.

Powinno brzmieć lepiej.